hw/ssi/sai.c

```diff
@@ -0,0 +1,93 @@
+/* Virtualization support for SAI.
+ *
+ * Copyright 2020 NXP
+ *
```

Comment:
We should be careful here. The instance of SAI integrated with imx8/imx8x indeed uses this register map. Anyhow, for imx8m the register map is a little bit different. 
See: https://github.com/thesofproject/sof/commit/182e62a198c3e700f6271ac986776f4ed8ace14a
Need to find a way to distinguish between when we compile this driver to be used with imx8 or imx8m